library/bonita

```diff
@@ -8,18 +8,18 @@ Architectures: amd64, arm64v8, ppc64le
 GitRepo: https://github.com/bonitasoft/bonita-distrib.git
 Directory: docker
 
-Tags: 2021.1, 7.12.1, 7.12
-GitFetch: refs/heads/docker/2021.1
-GitCommit: bfdd527629063b73f053320e50e1f6bfcd135d0a
-
 Tags: 2021.2-u0, 2021.2, 7.13.0, 7.13
 GitFetch: refs/heads/docker/2021.2
 GitCommit: a1d9ee5e31d38958aa553cc7f9d465f1151d902f
 
 Tags: 2022.1-u0, 2022.1, 7.14.0, 7.14
-GitFetch: refs/heads/master
+GitFetch: refs/heads/2022.1
 GitCommit: 694bf79347add872f8c6a4c0a7f5c3ef12c31dc8
 
-Tags: 2022.2-u0, 2022.2, 7.15.0, 7.15, latest
-GitFetch: refs/heads/master
+Tags: 2022.2-u0, 2022.2, 7.15.0, 7.15
+GitFetch: refs/heads/2022.2
 GitCommit: 21c0e51634e836feaf910e8e3d6e95de200e1814
```

Comment:
Any particular reason to keep these versions listed here?  Are multiple versions of Bonita actively supported at once?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Hi. The four latest versions are supported currently.